opencog/query/Distributed/DistSCM.cc

```diff
@@ -0,0 +1,230 @@
+/*
+ * DistSCM.cc
```

Comment:
This framework has essentially nothing to do with distributed queries at all; its a general distribution framework. it would be much better if you moved this all to the directory /opencog/persist  Yes, "persist" is a funny name for this, but that is where zmq sits, and its where hypertable sits, so its where this should sit as well.